include/seastar/core/reactor.hh

```diff
@@ -227,7 +227,7 @@ public:
         uint64_t fstream_read_bytes = 0;
         uint64_t fstream_reads_blocked = 0;
         uint64_t fstream_read_bytes_blocked = 0;
-        uint64_t fstream_read_aheads_discarded = 0;
+        uint64_t fstream_readaheads_discarded = 0;
```

Comment:
The name of the variable is internal, but later a `make_counter("fstream_reads_aheads_discarded")` call gives it a public name, and it will be strange if we change the variable to readaheads but not the counter name.
However, I do agree that readaheads is better than read_aheads :-(